Gemfile

```diff
@@ -2,18 +2,17 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: '0.19-stable' }
```

Comment:
This PR should branch off from the v0.20 branch in https://github.com/CodiTramuntana/decidim-salt-app/pull/35, which, by  the way, is already been merged into `master`.
Can you merge `master` then into this branch please?

Reply:
Done!!